### PR TITLE
Apply filter to solar ability colours.

### PR DESCRIPTION
**Pull request by ryan-rushton**
I just couldn't deal with it anymore 🤣

Screen Shot 2022-08-12 at 9 31 55 pm

Screen Shot 2022-08-12 at 9 29 00 pm

Screen Shot 2022-08-12 at 9 28 27 pm

### CODE REVIEW

**Review by robojumper — APPROVED** [review]
I like it! Would be good to include some short comment describing what this is actually trying to work around.

(Also maybe changelog, this is too funny to leave out)

**Comment by ryan-rushton** [comment]
Agreed, I was thinking I should comment it after I went to bed. Also added a changelog, feel free to update it to poke more fun at the isssue :D

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
   * Perk and mod tooltips for enhanced weapon traits and Exotic catalysts have unique styles to help them stand out.
   * The energy cost of armor mods is displayed within tooltips.
 * Fixed an issue where the energy meter on Ghosts was not displaying the amount of energy that had been used by inserted mods.
+* Solar class ability and jump icons have been filtered to match other solar abilities (we couldn't handle it anymore).
```
Comment (nev-r) [inline docs/CHANGELOG.md:8]:
```suggestion
* Solar class ability and jump icons have had their colors adjusted to match other solar abilities (we couldn't handle it anymore).
```

File: src/app/inventory/ItemIcon.tsx [hunk 2]
```diff
@@ -16,6 +16,27 @@ import pursuitComplete from 'images/highlightedObjective.svg';
 import { DimItem } from './item-types';
 import styles from './ItemIcon.m.scss';
 
+// These are the solar abilities that have statis colored images on their definitions.
+// To make them match the orange solar color (best we can) we apply a css filter to
+// change their color.
```
Comment (nev-r) [inline src/app/inventory/ItemIcon.tsx:21]:
```suggestion
// These solar abilities have stasis-colored icons in their Bungie-provided definitions.
// To make them match the orange solar color (best we can) we apply a css filter.
```

**Review by nev-r — APPROVED** [review]

**Pull request merged**